filebeat/module/mongodb/log/ingest/pipeline-json.yml

```diff
@@ -17,6 +17,12 @@ processors:
 - rename:
     field: mongodb.log.ctx
     target_field: mongodb.log.context
+- rename:
+    field: message
+    target_field: log.original
```

Comment:
Could we change this to `event.original` to be consistent with other modules/packages?

Reply:
@tetianakravchenko I [mentioned](https://github.com/elastic/beats/pull/26338#issuecomment-862286697) to use `log.original`, but `event.original` should be used instead.